src/renderer/components/snipping-tool.tsx

```diff
@@ -168,17 +213,21 @@ const SnippingTool = () => {
       return undefined;
     }
     if (chosenTool === Tool.pen) {
-      return { border: '2px solid ' + penColor };
+      const color = penColor.outline ? penColor.outline : penColor.rgbaColor;
+      return { border: '2px solid ' + color };
     } else if (chosenTool === Tool.highlight) {
-      return { border: '2px solid ' + highlightColor };
+      const color = highlightColor.outline ? highlightColor.outline : highlightColor.rgbaColor;
+      return { border: '2px solid ' + color };
     } else if (chosenTool === Tool.eraser) {
       return { border: '2px solid #008EFF' };
     }
     return undefined;
   };
 
-  const done = () => {
-    ipcRenderer.send('upload-snippet', screenSnippetPath);
+  const done = async () => {
+    const base64PngData = await getBase64PngData();
+    // const base64PngData = 'await getBase64PngData();';
```

Comment:
Required?

Reply:
1 - Good catch!

2 - Yes, that is true, artifacts will appear if you do fast movements, especially over itself in different directions. It is a limitation with svg paths. It is hard or impossible for me to do anything against.

3 - Haha, thank you :)